gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,5 @@
-#Fri Jul 27 18:14:34 BST 2018
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.10-all.zip
```

Comment:
I think @kittinunf had a reason why we couldn't do this (sorry I can't remember what it was)

Reply:
Good point. Maybe it's an issue with the [novoda/bintray-release](https://github.com/novoda/bintray-release) plugin. The README file says it's not compatible with Gradle 4.5 at the moment.

Reply:
@lucasvalenteds Actually, it mentioned about "Currently Gradle 4.5 doesn't work with Android projects." It does have own Kotlin DSL examples too.

Reply:
Thank you for the clarification, @iNoles. I'll look into a workaround.

Reply:
Yes, it has to be 4.4 at the moment or we get rid of `bintray-release`. But I would love to do that in the different PR.